src/common/components/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import styles from './SearchResults.module.scss'
+import { Container } from 'common/components/Container/index.js'
+
+export const SearchResults = () => {
+  return (
+    <div className={styles.searchResults}>
+      <Container>
```

Comment:
I'm not sure if this suggestion applies in this case, but if you want to style the `<Container>` with `styles.searchResults`, you can add a `className` prop to `<Container>`, e.g.:

```
// File: common/components/Container/Container.js

import React from 'react'
import classNames from 'classnames'

import styles from './Container.module.scss'

export const Container = ({ className, children }) => (
  <div className={classNames(styles.container, className)}>
    {children}
  </div>
)
```

Then, pass `className`: `<Container className={styles.searchResults}>`

Reply:
Thanks! That's good to know that I can apply module styles to a reusable component that way This time I do want to set it to the parent of the Container, though.